src/profile/types.mo

```diff
@@ -0,0 +1,14 @@
+import Hash "mo:stdlib/hash.mo";
+
+module {
+	public type PrincipalId = Hash.Hash;
```

Comment:
👍 
Although it might end up confusing people that read the public spec.

Reply:
Good point. We should either use the actual blob as the hash key or call this something different.